Modules/_sha3/sha3module.c

```diff
@@ -122,6 +122,28 @@
 #define SHA3_squeeze Keccak_HashSqueeze
 #define SHA3_copystate(dest, src) memcpy(&(dest), &(src), sizeof(SHA3_state))
 
+typedef struct {
+    PyTypeObject *sha3_224_type;
+    PyTypeObject *sha3_256_type;
+    PyTypeObject *sha3_384_type;
+    PyTypeObject *sha3_512_type;
+    PyTypeObject *shake_128_type;
+    PyTypeObject *shake_256_type;
```

Comment:
nitpick: the whole file uses the order "sha3, keccak, shake". Just please move shake at the end of the structure :-)